.changeset/tender-rings-drive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/alphavantage-test-adapter': minor
```

Comment:
Since this is a new EA could you bump this to `major`

Reply:
Done